> Except if these are used in privileged unit tests, no?

True, yes. Thanks @pchaigno!

Runtime tests failed because of a merge conflict between #13107 and #13036

```
07:37:58  Failed to compile test:
07:37:58  
07:37:58  # github.com/cilium/cilium/test/helpers
07:37:58  helpers/kubectl.go:667:38: too many arguments in call to kub.GetCiliumPods
07:37:58  	have (string)
07:37:58  	want ()
07:37:58  note: module requires Go 1.15
```

https://jenkins.cilium.io/job/Cilium-PR-Runtime-4.9/1750/

Sent #13124 with a fix.

--- a/pkg/fqdn/util_test.go
+++ b/pkg/fqdn/util_test.go
@@ -17,12 +17,8 @@
 package fqdn
 
 import (
-	"encoding/json"
-	"fmt"
 	"net"
-	"strings"
 
-	"github.com/cilium/cilium/pkg/policy/api"
 	"github.com/miekg/dns"
 )
 
@@ -42,54 +38,6 @@ var (
 				net.ParseIP("001:4998:58:1836::10")}},
 	}
 )
-
-func makeRule(key string, dnsNames ...string) *api.Rule {
-	matchNames := []string{}
-	for _, name := range dnsNames {
-		matchNames = append(matchNames,
-			fmt.Sprintf(`{"matchName": "%s"}`, dns.Fqdn(name)))
-	}
-
-	rule := `{`
-	if key != "" {
-		rule += fmt.Sprintf(`"labels": [{ "key": "%s" }],`, key)
-	}
-	rule += fmt.Sprintf(`"endpointSelector": {
-    "matchLabels": {
-      "class": "xwing"
-    }
-  },
-  "egress": [
-    {
-      "toFQDNs": [
-      %s
-      ]
-    }
-  ]
-}`, strings.Join(matchNames, ",\n"))
-	//fmt.Print(rule)
-	return mustParseRule(rule)
-}
-
-func parseRule(rule string) (parsedRule *api.Rule, err error) {
-	if err := json.Unmarshal([]byte(rule), &parsedRule); err != nil {
-		return nil, err
-	}
-
-	if err := parsedRule.Sanitize(); err != nil {
-		return nil, err
-	}
-
-	return parsedRule, nil
-}
-
-func mustParseRule(rule string) (parsedRule *api.Rule) {
-	parsedRule, err := parseRule(rule)
-	if err != nil {
-		panic(fmt.Sprintf("Error parsing FQDN test rules: %s", err))
-	}
-	return parsedRule
-}
 
 // LookupDNSNames is a wrappable dummy used by the tests. It counts the number
 // of times a name is looked up in lookups, and uses ipData as a source for the

